### PR TITLE
Media section

### PR DESCRIPTION
**Pull request by AntonioMateoGomez**
This is a first approach of the media section still WIP since needs:

- [x] Responsive mode

- [x] search and media tabs behaviour in small size view

- [x] favicon and other images

- [x] Also as a doubt we must to know the behaviour when a post contains several tags.

### CODE REVIEW

File: _includes/_head.html [hunk 1]
```diff
@@ -31,6 +32,7 @@
     {% include _google-tag-manager.html %}
   {% endif %}
 
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
```
Comment (calvellido) [inline _includes/_head.html]: Could we please replace jQuery for native code?

File: js/hovers.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Auxiliar function to identify the current icon and span objects when mapping
+function getCurrentId(id) {
+
+  let iconId = id;
```
Comment (calvellido) [inline js/hovers.js]: No need for `let` in here. Use `const`.

File: js/hovers.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      icon: iconObject,
+      span: iconSpanObject,
+  };
+}
+
+function losingFocus(e){
+
+  Array.prototype.map.call(iconFilters, function(obj) {
+
+    let {icon, span} = getCurrentId(obj.id);
```
Comment (calvellido) [inline js/hovers.js]: Nor `icon`, nor `span` are being modified. Use `const`.

File: js/hovers.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  return {
+      icon: iconObject,
+      span: iconSpanObject,
+  };
+}
+
+function losingFocus(e){
+
+  Array.prototype.map.call(iconFilters, function(obj) {
```
Comment (calvellido) [inline js/hovers.js]: This is the same behaviour we've seen previously. Please change to a usual way of calling `map`in an Array,

File: js/hovers.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      span.style.opacity = "1";
+    }
+  });
+}
+
+function checkActiveTag() {
+
+  Array.prototype.map.call(iconFilters, function(obj) {
+
+    let {icon, span} = getCurrentId(obj.id);
```
Comment (calvellido) [inline js/hovers.js]: No need for `let` in here. Use `const`.

File: js/hovers.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    } else {
+      icon.style.opacity = "0.4";
+      span.style.opacity = "0.4";
+    }
+  });
+}
+
+function applyingFocus(e) {
+
+  Array.prototype.map.call(iconFilters, function(obj) {
```
Comment (calvellido) [inline js/hovers.js]: This is the same behaviour we've seen previously. Please change to a usual way of calling `map` in an Array.

File: js/hovers.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    } else {
+      icon.style.opacity = "1";
+      span.style.opacity = "1";
+    }
+  });
+}
+
+function checkActiveTag() {
+
+  Array.prototype.map.call(iconFilters, function(obj) {
```
Comment (calvellido) [inline js/hovers.js]: This is the same behaviour we've seen previously. Please change to a usual way of calling `map` in an Array.

File: js/hovers.js [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+      span.style.opacity = "0.4";
+    }
+  });
+}
+
+function applyingFocus(e) {
+
+  Array.prototype.map.call(iconFilters, function(obj) {
+
+    let {icon, span} = getCurrentId(obj.id);
```
Comment (calvellido) [inline js/hovers.js]: Nor `icon`, nor `span` are being modified. Use `const`.

File: _includes/_tab.html [hunk 9]
```diff
@@ -1,49 +1,67 @@
...
-                    $("#results-container").show();
-                  } else {
-                    $("#results-container").hide();
-                  }
-                });
-              </script>
-          </nav>
-      </div>
+          function searchControl(input) {
+            console.log('input: ', input);
```
Comment (calvellido) [inline _includes/_tab.html]: No loggin' please :pensive:

Comment (AntonioMateoGomez) [reply _includes/_tab.html]: So sorry

**Pull request merged**